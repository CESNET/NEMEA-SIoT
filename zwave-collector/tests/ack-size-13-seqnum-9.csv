time TIMESTAMP,uint8 CHANNEL,bytes FRAME
2019-07-29T17:38:45.935,2,fe90821d0353090d012503ff8200
