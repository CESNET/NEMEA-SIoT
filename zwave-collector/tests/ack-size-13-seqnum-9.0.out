3,2019-07-29T17:38:45.935,4270883357,1,2,0,1,0,0,9,13,0,"","",,,"","Ack"
